tcmur_cmd_handler.c

```diff
@@ -816,12 +842,18 @@ static int write_verify_init(struct tcmulib_cmd *origcmd, size_t length)
 {
 	struct tcmulib_cmd *readcmd;
 	struct write_verify_state *state;
+	struct tcmur_cmd_state *read_rcmd_state;
+	struct tcmur_cmd_state *orig_rcmd_state = tcmu_cmd_get_private(origcmd);
 	int i;
 
+
+	read_rcmd_state = calloc(1, sizeof(*read_rcmd_state));
+	if (!read_rcmd_state)
+		goto out;
 	readcmd = calloc(1, sizeof(*readcmd));
 	if (!readcmd)
-		goto out;
-	readcmd->cmdstate = origcmd;
+		goto free_rcmd;
+	read_rcmd_state->cmdstate = origcmd;
```

Comment:
It seems the same here forgot to set read_rcmd_state to readcmd.

Reply:
Yeah, thanks.